src/utils/schema-utils.js

```diff
@@ -19,7 +19,9 @@ export function getTypeInfo(schema, options = { includeNulls: false }) {
   } else if (schema.type) {
     const arraySchema = Array.isArray(schema.type) ? schema.type : (typeof schema.type === 'string' ? schema.type.split('┃') : schema.type);
     dataType = Array.isArray(arraySchema) ? arraySchema.filter((s) => s !== 'null' || options.includeNulls).join('┃') : schema.type;
-    dataType = dataType.replace('string', typeof schema.const !== 'undefined' && 'const' || schema.enum && 'enum' || schema.format || 'string');
+    ['string', 'number', 'boolean'].forEach(type => {
```

Comment:
According to the spec, it has to be a `string`: https://spec.openapis.org/oas/latest.html#serverVariableEnum I'm okay with adding number support here, but it doesn't make sense for boolean, so let's pull that one out.

```suggestion
    ['string', 'number'].forEach(type => {
```